src/greeting.rs

```diff
@@ -65,6 +67,23 @@ pub fn welcome(settings: &Settings) {
     println!("{}", paint(r#" \__, |\__,_|_|  \__,_|_| |_|\__| "#, true_color));
     println!("{}", paint(r#"    |_|                           "#, true_color));
     println!();
+
+    // Print current version and, if available, first 8 characters of the git commit hash
+    let git_commit_info = get_git_commit_id()
+        .map(|git_commit| format!(
+            ", {} {}",
+            "build:".truecolor(134, 186, 144),
+             &git_commit[..min(8, git_commit.len())].bold().truecolor(82, 139, 183)
```

Comment:
Check if `true_color == true` is required here. I propose to reuse `fn paint(text: &str, true_color: bool)` like in dqrant logo printing